pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -424,6 +426,9 @@ func (ctrl *PersistentVolumeController) syncUnboundClaim(claim *v1.PersistentVol
 					// This should never happen because someone had to remove
 					// AnnBindCompleted annotation on the claim.
 					klog.V(4).Infof("synchronizing unbound PersistentVolumeClaim[%s]: volume already bound to different claim %q by controller, THIS SHOULD NEVER HAPPEN", claimToClaimKey(claim), claimrefToClaimKey(volume.Spec.ClaimRef))
+					claimMsg := fmt.Sprintf("volume %q already bound to a different claim.", volume.Name)
```

Comment:
It seems the two calls to ctrl.eventRecorder.Event have the same arguments.

Note the comment on line 426.
How about changing the message as in the following ?
```
                                        claimMsg := fmt.Sprintf("should not have happened: volume %q already bound to a different claim.", volume.Name)
```
This would allow users to distinguish the two events added.